pkg/cmd/codespace/ssh.go

```diff
@@ -398,10 +398,17 @@ func newConfigCmd(app *App) *cobra.Command {
 		Use:   "config [-c codespace]",
 		Short: "Write OpenSSH configuration to stdout",
 		Long: heredoc.Docf(`
-			The config command generates ssh connection configuration in OpenSSH format.
+			The config command generates per-codespace ssh configuration in OpenSSH format.
+
+			Including this configuration in ~/.ssh/config simplifies integration with other tools
```

Comment:
Ah, neat, now I want to use it. :)

Nitpick: avoid repetition of "integrate": 
s/simplifies integration with/improves the user experience of/